Dockerfile

```diff
@@ -138,6 +145,31 @@ ENV DEBIAN_FRONTEND=noninteractive \
     RCON_PORT=25575 \
     REGION= \
     USEAUTH=true \
-    BAN_LIST_URL=https://api.palworldgame.com/api/banlist.txt
+    BAN_LIST_URL=https://api.palworldgame.com/api/banlist.txt \
+    ### Webhook-settings
+    WEBHOOK_ENABLED=false \
+    WEBHOOK_URL= \
+    WEBHOOK_START_TITLE="Server is starting" \
+    WEBHOOK_START_DESCRIPTION="The gameserver is starting" \
+    WEBHOOK_START_COLOR="2328576" \
+    WEBHOOK_STOP_TITLE="Server has been stopped" \
+    WEBHOOK_STOP_DESCRIPTION="The gameserver has been stopped" \
+    WEBHOOK_STOP_COLOR="7413016" \
+    WEBHOOK_INFO_TITLE="Info" \
+    WEBHOOK_INFO_DESCRIPTION="This is an info from the server" \
+    WEBHOOK_INFO_COLOR="2849520" \
+    WEBHOOK_UPDATE_TITLE="Updating server" \
+    WEBHOOK_UPDATE_DESCRIPTION="Server is being updated" \
+    WEBHOOK_UPDATE_COLOR="2849520" \
+    WEBHOOK_UPDATE_VALIDATION_TITLE="Updating and validating server" \
+    WEBHOOK_UPDATE_VALIDATION_DESCRIPTION="Server is being updated and validated" \
+    WEBHOOK_UPDATE_VALIDATION_COLOR="2849520"
+
+
+EXPOSE 8211/udp
+EXPOSE ${RCON_PORT}/tcp
```

Comment:
Doing this dynamically makes no sense whatsoever and removes readibility. It should be set to 25575 and thats it.

As the documentation says:
The EXPOSE instruction doesn't actually publish the port. It functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published. To publish the port when running the container, use the -p flag on docker run to publish and map one or more ports, or the -P flag to publish all exposed ports and map them to high-order ports.

By default, EXPOSE assumes TCP. You can also specify UDP

Source: https://docs.docker.com/engine/reference/builder/#expose